### PR TITLE
Update 2018-00-00-havanar.md

### PR DESCRIPTION
**Pull request by TanjaKec**
Final version approved by Julie and Ziv

### CODE REVIEW

**Comment by missaugustina** [comment]
I restarted the build, it looks like it failed during setup (apt-get etc...)

File: content/blog/2018-00-00-havanar.md [hunk 1]
```diff
@@ -9,29 +9,30 @@ title: HavanaR Workshop 2018
 slug: havanar
 ---
 
-The first meeting of R users in Cuba, HavanaR! club 2018, will take place From 29-10-2018 to 02-11-2018. The workshop is organized jointly by the Group of Probability and Statistics of Universidad de La Habana, the >eR-Biostat initiavle, I-Bioest, Hasselt University, Belgium and the R Forward taskforce.
+The first meeting of R users in Cuba, HavanaR! Club 2018, will take place From 29-10-2018 to 02-11-2018. The workshop is organized jointly by the Group of Probability and Statistics of Universidad de La Habana, the >eR-Biostat initiative, the Center for Statistics, CenStat & I-Biostat, Hasselt University, Belgium and the R Forward taskforce.
```
Comment (hturner) [inline content/blog/2018-00-00-havanar.md]: On our own website the taskforce can be referred to simply as "Forwards". (This is the official name, but often the R is added to clarify what we're about, especially in a general environment such as Twitter, vs a specific R environment, such as useR!).

File: content/blog/2018-00-00-havanar.md [hunk 2]
```diff
@@ -9,29 +9,30 @@ title: HavanaR Workshop 2018
...
 
-The workshop is free but registration in mandatory. Registration can be done by email to erbiostat@gmail.com or cbouza2002@gmail.com.
+The workshop is free but registration in mandatory. To register please send an email to erbiostat@gmail.com or cbouza2002@gmail.com.
 
  
 
-More details and materials  will be available online in https://er-biostat.github.io/Courses/ Updates about the workshop will communicate in FACEBOOK (https://www.facebook.com/ER-BioStat-1463845487001786/) and Twitter (@erbiostat).
\ No newline at end of file
+More details and workshop materials  will be available online at https://er-biostat.github.io/Courses/ 
+All updates about the workshop will be anounced on facebook page (https://www.facebook.com/ER-BioStat-1463845487001786/) and Twitter (@erbiostat).
```
Comment (hturner) [inline content/blog/2018-00-00-havanar.md]: "the Facebook page"

**Review by hturner — CHANGES_REQUESTED** [review]
We will need a banner image for the blog as well - I don't know if Ziv has anything? Else maybe @missaugustina can advise?

**Comment by hturner** [comment]
I'm not sure our build works with pull requests from accounts outside the Forwards GitHub organisation. There's no harm in accepting a pull request to a branch though as that won't get pushed to production. So maybe when @TanjaKec has made the minor corrections I suggested, this can be accepted and we can take it from there? @TanjaKec should be part of the org, I will chase that up.

**Comment by hturner** [comment]
Okay merging this into branch for finishing off

**Comment by missaugustina** [comment]
Looks good to me, sorry for the late response!

**Comment by missaugustina** [comment]
Looking at this now, sorry for the delay!

**Review by missaugustina — COMMENTED** [review]
Agree with Heather's comments, I think just those small edits and this should be good.

**Pull request merged**